evaluator/evaluator.py

```diff
@@ -270,31 +270,40 @@ def evaluate_vmaas(self, system_platform, cur, loop=None):
         new_cves = set()
         mitigated_cves_ids = set()
         unmitigated_cves_ids = set()
+        inc_cves_ids = set()
+        dec_cves_ids = set()
 
         for cve in reported_cves:
             if cve in system_cves_map:
                 unprocessed_cves.discard(cve)
                 if system_cves_map[cve]['when_mitigated']:
                     # it was mitigated, now its not
                     unmitigated_cves_ids.add(system_cves_map[cve]['cve_id'])
+                    if system_cves_map[cve]['rule_id'] is None:  # unmitigated only if there wasn't rule hit before
+                        inc_cves_ids.add(system_cves_map[cve]['cve_id'])
             else:
                 new_cves.add(cve)
 
+        rule_cves = 0
```

Comment:
counter for system CVEs coming only from rules results as there might be some CVEs reported only by rules engine